swaintek/gulpfile.js

```diff
@@ -0,0 +1,23 @@
+var gulp = require ('gulp');
+var mocha = require('gulp-mocha');
+var eslint = require('gulp-eslint')
+
+gulp.task('mocha', function () {
+  return gulp.src(['test/greet_test.js'], {read: false})
+    .pipe(mocha());
+});
+
+gulp.task('lint', function () {
+  return gulp.src(['**/*.js', '!node_modules'])
+    .pipe(eslint({
+      rules: {
```

Comment:
you can add something like this to your call to eslint to fix ES lint errors:
.pipe(eslint({
      parserOptions: {
        "ecmaVersion": 6
      }...